wandb/sdk/interface/interface.py

```diff
@@ -70,6 +72,18 @@ class FilesDict(TypedDict):
 logger = logging.getLogger("wandb")
 
 
+def file_type_to_enum(file_type: "FileType") -> "pb.FilesItem.FileType.V":
+    if file_type == "media":
+        enum = pb.FilesItem.FileType.MEDIA
+    elif file_type == "wandb":
+        enum = pb.FilesItem.FileType.WANDB
+    elif file_type == "artifact":
+        enum = pb.FilesItem.FileType.ARTIFACT
+    else:
+        enum = pb.FilesItem.FileType.OTHER
+    return enum
```

Comment:
it's more idiomatic to return from each branch instead

Reply:
yeah i can fix it